less/angular/pages/home.less

```diff
@@ -534,3 +528,517 @@
+  }
+  color: @privacyred;
+  padding: 5rem 0 6rem;
+  text-align: center;
+
+  h2 {
+    margin-bottom: 2rem;
+  }
+
+  h4 {
+    color: @grey;
+    font-weight: 200;
+    margin-bottom: 2rem;
+  }
+
+  .btn {
+    border: transparent;
+    border-radius: 0.3rem;
+    box-shadow: 0 0.4rem 0 @darkgreen, inset 0 0.1rem 0 rgba(255, 255, 255, 0.5);
+    font-weight: 600;
+    letter-spacing: 0.1rem;
+    line-height: 1.42857;
+    padding: 1.3rem 1.5rem 1.1rem 1.5rem;
+    position: relative;
+    text-transform: uppercase;
+
+    &:hover, &:focus {
+      box-shadow: 0 0.2rem 0 @darkgreen, inset 0 0.1rem 0 rgba(255, 255, 255, 0.5);
+      background-color: @brand-primary;
+      top: 0.2rem;
+    }
+    &:active {
+      box-shadow: inset 0 0.1rem 0.1rem rgba(0, 0, 0, 0.2);
+      top: 0.4rem;
+    }
+  }
+}
+
+.privacy-eye {
+  height: 7.5rem;
+  margin: 1.5rem auto;
+  position: relative;
+  z-index: 300;
+  width: 7.5rem;
+
+  h6 {
+    font-size: 0.9em;
+    font-weight: 600;
+    margin: 0;
+    position: relative;
+    top: -0.2rem;
+  }
+
+  p {
+    font-size: 0.7em;
+    font-weight: 400;
+    position: relative;
+    top: 0.1rem;
+  }
+
+  a {
+    &:link, &:visited, &:hover, &:active, &:focus {
+      color: @privacyred;
+    }
+  }
+
+  svg {
+    height: 7.5rem;
+    position: relative;
+    left: -0.4rem;
+    top: -2.2rem;
+    width: 100%;
+  }
+
+  .front {
+    background-color: @privacyred;
+    float: none;
+    height: inherit;
+    position: absolute;
+    top: 0;
+    left: 0;
+    text-align: center;
+    width: inherit;
+
+    &::before, &::after {
+      z-index: -1;
+    }
+
+    &::before {
+      border-bottom: 2.15rem solid @privacyred;
+      top: -2rem;
+    }
+
+    &::after {
+      border-top: 2.15rem solid @privacyred;
+      bottom: -2rem;
+      left: 0;
+    }
+  }
+  .back {
+    background-color: @privacygrey;
+    float: none;
+    left: 0;
+    height: inherit;
+    position: absolute;
+    top: 0;
+    width: inherit;
+    z-index: -1;
+
+    &::before, &::after {
+      left: 0;
+      z-index: -2;
+    }
+
+    &::before {
+      border-bottom: 2.15rem solid @privacygrey;
+      top: -2rem;
+    }
+
+    &::after {
+      border-top: 2.15rem solid @privacygrey;
+      bottom: -2rem;
+    }
+  }
+
+  .front, .back {
+    width: 7.5rem;
+    height: 4.33rem;
+    margin: 2.165rem 0;
+    border-style: solid;
+    border-width: 0;
+    border-left-color: transparent;
+    border-right-color: transparent;
+    border-radius: 0.4rem;
+    &::before, &::after {
+      content: '';
+      background-color: transparent;
+      border-left: 3.75rem solid transparent;
+      border-right: 3.75rem solid transparent;
+      left: 0;
+      height: 0;
+      position: absolute;
+      width: 0;
+    }
+  }
+}
+
+.privacy-svg {
+  display: none;
+  &.show {
+    display: block;
+  }
+}
+.privacy-gif {
+  display: block;
+  &.hide {
+    display: none;
+  }
+}
+
+
+.privacy-eye {
```

Comment:
I looks like there is another .privacy-eye {} ruling on line 583. They can likely be merged if we want.

Not a huge deal.